We don't build/install interfaces by upstream's implicit rules.

This patch is used on two places; postgresql.spec and libecpg.spec -- keep those
in sync!

Related: rhbz#1618698

diff --git a/src/Makefile b/src/Makefile
index bcdbd95..4bea236 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -20,7 +20,6 @@ SUBDIRS = \
 	backend/utils/mb/conversion_procs \
 	backend/snowball \
 	include \
-	interfaces \
 	backend/replication/libpqwalreceiver \
 	backend/replication/pgoutput \
 	fe_utils \
diff --git a/src/Makefile.global.in b/src/Makefile.global.in
index b75bafd..ddd381b 100644
--- a/src/Makefile.global.in
+++ b/src/Makefile.global.in
@@ -576,7 +576,7 @@ endif
 # How to link to libpq.  (This macro may be used as-is by backend extensions.
 # Client-side code should go through libpq_pgport or libpq_pgport_shlib,
 # instead.)
-libpq = -L$(libpq_builddir) -lpq
+libpq = -lpq
 
 # libpq_pgport is for use by client executables (not libraries) that use libpq.
 # We want clients to pull symbols from the non-shared libraries libpgport
@@ -619,7 +619,6 @@ endif
 # Commonly used submake targets
 
 submake-libpq: | submake-generated-headers
-	$(MAKE) -C $(libpq_builddir) all
 
 submake-libpgport: | submake-generated-headers
 	$(MAKE) -C $(top_builddir)/src/port all
-- 
2.21.0

